### DISCUSSION TITLE
[Serenity BDD reporter] - Is there a way to reduce the size of the screenshot generated by serenity BDD reporter

### DISCUSSION THREAD

**Discussion by enliltom004** [serenity-js/serenity-js#1635] [How to... ?]
Having issues taking screenshot of a test which is running in a cloud platform . It shows the image is too large and it cannot process it . 

Can we pass any other settings to reduce the image size .

Framework : wdio/serenity 
Node : 16.16.0
"@serenity-js/serenity-bdd": "^2.33.10",

```
   crew: [
      ArtifactArchiver.storingArtifactsAt('./target/site/serenity'),
      Photographer.whoWill(TakePhotosOfFailures),
      ConsoleReporter.forDarkTerminals(),
      new SerenityBDDReporter(),
    ],
```

**Comment by jan-molak** [comment]
Hi @enliltom004, the size of the screenshot depends on its resolution, and its resolution depends on the size of the browser window.

Depending on the cloud platform you use, browser window size might need to be configured differently so its best to check the documentation of the platform you use.

In most cases, you'll need to specify window size as part of your [WebdriverIO configuration](https://webdriver.io/docs/options/#capabilities)
```javascript
{
  browserName: 'chrome',
  'goog:chromeOptions': {
    args: [ 'window-size=1920,1080' ],
  }
}
```
Hope this helps!

**Comment by enliltom004** [comment]
Hi @jan-molak , Thank you for your reply . Unfortunately the issue is with mobile browsers , and the platform wont allow custom resolution for mobile browsers .

**Comment by jan-molak** [comment]
Hi @enliltom004 and thanks for explaining. Could you advise what platform you're using and what's your WebdriverIO configuration around browser capabilities?
